plenum/test/view_change/slow_nodes/test_view_change_gc_all_nodes_random_delay.py

```diff
@@ -0,0 +1,127 @@
+from itertools import combinations
+
+from stp_core.loop.eventually import eventually
+
+from plenum.test import waits
+from plenum.test.helper import sendRandomRequests, \
+        send_reqs_to_nodes_and_verify_all_replies, \
+        waitForSufficientRepliesForRequests, \
+        checkViewNoForNodes, \
+        check_last_ordered_3pc
+from plenum.test.delayers import delay_3pc_messages, \
+        reset_delays_and_process_delayeds
+from plenum.test.view_change.helper import ensure_view_change_complete
+from plenum.test.pool_transactions.conftest import clientAndWallet1, \
+        client1, wallet1, client1Connected, looper
+
+
+TestRunningTimeLimitSec = 300
+
+
+def ensure_nodes_last_ordered_3pc(nodes, last_ordered_3pc):
+    for n1, n2 in combinations(nodes, 2):
+        lst_3pc = check_last_ordered_3pc(n1, n2)
+
+    assert lst_3pc == last_ordered_3pc
+
+
+def ensure_nodes_requests_size(nodes, size):
+    for node in nodes:
+        assert len(node.requests) == size
+
+
+def test_view_change_gc_in_between_3pc_all_nodes_random_delays(
+        looper, txnPoolNodeSet, wallet1, client):
+    """
+    Test that garbage compares the whole 3PC key (viewNo, ppSeqNo) and
```

Comment:
I think it's about queues cleared out in `_gc` which belong to node (an example of this is `requests`). So, the issue is when there is a request in this queue with key (1,1), but `_gc` is called for pp_seq_no `1` assuming 'old' view_no (0,1).